android_library/app/src/main/java/ch/uzh/supersede/hostapplication/StartActivity.java

```diff
@@ -55,7 +55,7 @@ public void onButtonClicked(View view){
             }
             case R.id.trigger_specific_pull_config_v1_id21_button:{
                 ConfigurationRequestWrapper wrapper = new ConfigurationRequestWrapperBuilder(StartActivity.this,FeedbackActivity.class)
-                        .withApplicationId(9)
+                        .withApplicationId(14)
```

Comment:
Ich nehme an, dass das die ID ist, die von Ronnie und co. verwendet wurde? Ist nämlich eine ID die wir laut Mail nicht nutzen dürfen. Sollte mit dem Orchestrator stub ja auch nicht mehr nötig für Testzwecke sein, oder?

Reply:
True that, die methode holt eh nur die config, da die aber nicht matcht wird keine je gesetzt. Müssen die id noch aus dem appdata purgen, ronnie hat sie da hinterlegt. Prinzipiell können wir sie aber ignorieren mit dem stub, jedoch ist das feedback-post noch als todo, das gibts noch nicht